src/main/java/eu/ai4work/sws/model/generic/SlidingDecisionResponse.java

```diff
@@ -0,0 +1,11 @@
+package eu.ai4work.sws.model.generic;
+
+import lombok.Data;
+import java.util.Map;
+
+@Data
+public class SlidingDecisionResponse {
+    private String decisionStatus;
```

Comment:
Better use the enum type:
```suggestion
    private SlidingDecisionStatus decisionStatus;
```